superset-frontend/src/explore/components/ExploreViewContainer.jsx

```diff
@@ -458,7 +459,7 @@ function ExploreViewContainer(props) {
           #app {
             flex-basis: 100%;
             overflow: hidden;
-            height: 100vh;
+            height: 100%;
```

Comment:
The `#app` should account for `#app-menu` height. That's why I change it to percentage and set the `body` to `100vh`.